src/App.js

```diff
@@ -1,16 +1,44 @@
-import React from 'react';
+import React, {useState} from 'react';
 import './App.css';
 import chatMessages from './data/messages.json';
+import ChatLog from './components/ChatLog';
+
+
 
 const App = () => {
+  const [messageData, setMessageData] = useState(chatMessages);
+  let [totalLikes, setTotalHeart] = useState(0);
+
+  const updateText = (updateText) => {
+    const messages = messageData.map((message) => {
+      if (message.id === updateText.id) {
+        return updateText;
+      } else {
+        return message;
+      }
+    });
+    setMessageData(messages);
+  };
+
+  const updateTotalLikes = (bool) => {
+    bool ? setTotalHeart((totalLikes += 1)) : setTotalHeart((totalLikes -= 1));
+  };
   return (
-    <div id="App">
+    <div id='App'>
       <header>
-        <h1>Application title</h1>
+        <h1>Chat with {chatMessages[0].sender} and {chatMessages[1].sender}</h1>
+        <h2>{totalLikes > 0 && `${totalLikes} ❤️s`}</h2>
```

Comment:
Love the use of the `&&` to check if the total likes banner should be shown or not! ❤️‍🔥